countries/app.rb

```diff
@@ -0,0 +1,53 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'sinatra/activerecord'
+
+set :database, { adapter: 'sqlite3', database: 'test.db' }
+
+class Question < ActiveRecord::Base
+    
+end
+
+get '/questions' do
+    @questions = Question.all
+    erb :questions
+end
+
+get '/questions/new' do
+    erb :new
+end
+
+post '/questions' do
+    question = Question.new
+    question.question = params[:question]
+    question.answer = params[:answer]
+    question.flag = params[:flag]
+    question.save
+    redirect '/questions'
+end
+
+get '/questions/edit' do
+    @question = Question.find(params [:id])
+    erb :edit
+end
+
+post '/questions/edit' do
+    question = Question.find(params [:id])
+    question.question = params [:question]
+    question.answer = params[:answer]
+    question.flag = params[:flag]
+    question.save
+    redirect '/questions'
+end
+
+get '/guess' do
+    @questions = Question.all
+    index = rand (@questons.length)
+    @question = questions[index]
```

Comment:
I know this might seems weird...but writing `questions` is not actually the same as `@questions`! My personal preference is to *always* use the local variable (here: `questions`) rather than the instance variable (`@questions`) unless you *need* to use it the instance variable outside of the method (say in a view or in another method).